### DISCUSSION TITLE
Option to Migrate Pterodactyl Data to Pelican

### DISCUSSION THREAD

**Discussion by SirOMGitsYOU** [pelican-dev/panel#13] [Ideas]
I think it would be a good option especially since most people will move directly from Ptero to Pelican if there is going to be more active development over here, I think it could save a lot of time if there was some sort of import/migration process to take the ptero data and filter it and convert it into pelican compatible data

**Comment by notAreYouScared** [comment]
This will be supported.

**Comment by crackscout123** [comment]
But when will it be supported? Is there any info about this feature getting released soon? 
Because the FAQ still says "You'll receive an easy guide to migrate from Pterodactyl to Pelican. It will be at least as easy as upgrading was originally." since the website in published. I really want to switch to pelican but i dont want to risk breaking the current system. I cant afford a big downtime /:

**Reply by Boy132** [reply]
The migration guide will be written when there is a stable release. (no ETA)

**Comment by crackscout123** [comment]
Alright, I’ll wait
thanks
